spec/features/navigation_spec.rb

```diff
@@ -100,9 +109,9 @@
     end
 
     # it "I can see Admin Dashboard when logged in as an merchant" do
```

Comment:
Let's leave this test in in case it's still needed, but I think we'll be able to take it out eventually.  Likely Grant's handling of US4 will make this irrelevant.

Reply:
Awesome - when we go a Navigation full PR we can confirm we can remove this then :)